### PR TITLE
Scripts/Spells : Paladin Spell BlindingLight

### PR DESCRIPTION
**Pull request by Hadifaren**
Spell Paladin Blinding Light 115750 Effect

**Target branch(es):** 3.3.5/master
- [x] master

**Tests performed:** (Does it build, tested in-game, etc.)
Tested Work in Game 7.1.5

### CODE REVIEW

File: src/server/scripts/Spells/spell_paladin.cpp [hunk 1]
```diff
@@ -334,6 +335,28 @@ class spell_pal_blessing_of_faith : public SpellScriptLoader
...
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_PALADIN_BLINDING_LIGHT_EFFECT });
+    }
+
+    void HandleDummy(SpellEffIndex /*effIndex*/)
+    {
+        if (Unit* target = GetHitUnit())
+            GetCaster()->CastSpell(target, SPELL_PALADIN_BLINDING_LIGHT_EFFECT, true);
```
Comment (joschiwald) [inline src/server/scripts/Spells/spell_paladin.cpp]: what happens if the effect breaks on damage? should this aura still remains on target?

Comment (Hadifaren) [reply src/server/scripts/Spells/spell_paladin.cpp]: no it removes auto as i saw on creature

Comment (Hadifaren) [reply src/server/scripts/Spells/spell_paladin.cpp]: i did not test as i dont have good creatures to do on or player to test with

Comment (joschiwald) [reply src/server/scripts/Spells/spell_paladin.cpp]: forget what i said

**Comment by joschiwald** [comment]
pls add the sql too

**Pull request merged**